README.rst

```diff
@@ -19,9 +19,8 @@ Functionality
    expressions after the entire file has been read.  This enables support for
    first-class forward references that stub files use.
 
-The latter should ideally be merged into ``flake8`` as the integration is
-currently pretty brittle (might break with future versions of ``pyflakes``,
-``flake8``, or due to interactions with other overly clever plugins).
```

Comment:
I think it would be good to mention the incompatibility in README too. Many people have been confused by how this plugin doesn't work if you have other seemingly unrelated plugins installed.